testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -754,6 +757,20 @@ public void testShowCreateInformationSchemaTable()
                         ")");
     }
 
+    @Test
+    public void testCreateTableNotAllowedInTransaction()
+    {
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE) && !hasBehavior(SUPPORTS_MULTI_STATEMENT_WRITES));
```

Comment:
We should also cover a case where `hasBehavior(SUPPORTS_MULTI_STATEMENT_WRITES)` to see that connector implement contract properly.

Reply:
Done